src/net_processing.cpp

```diff
@@ -146,6 +147,65 @@ static constexpr uint32_t MAX_GETCFILTERS_SIZE = 1000;
 static constexpr uint32_t MAX_GETCFHEADERS_SIZE = 2000;
 /** the maximum percentage of addresses from our addrman to return in response to a getaddr message. */
 static constexpr size_t MAX_PCT_ADDR_TO_SEND = 23;
+/** Static component of the salt used to compute short txids for transaction reconciliation. */
+static const std::string RECON_STATIC_SALT = "Tx Relay Salting";
+/** Used to convert a floating point reconciliation coefficient q to an int for transmission. */
+static constexpr double Q_PRECISION{2 << 12};
+/** Default coefficient used to estimate set difference for tx reconciliation. */
+static constexpr double DEFAULT_RECON_Q = 0.02;
+/**
+ * When considering whether we should flood to an outbound connection supporting reconciliation,
+ * see how many outbound connections are already used for flooding. Flood only if the limit is not reached.
+ * It helps to save bandwidth and reduce the privacy leak.
+ */
+static constexpr uint32_t MAX_OUTBOUND_FLOOD_TO = 8;
+/**
+ * Maximum number of elements to store in the reconciliation set.
+ * Used to bound the memory use. If the limit is reached, new transactions are not added and
+ * forgotten w.r.t. relaying to that peer.
+ * Sets the bound on the following objects:
+ * - reconciliation set
+ * - reconciliation set snapshot
+ * - reconciliation short-full id mapping
+ */
+static const unsigned int MAX_RECON_SET = 10000;
+/**
+ * Interval between sending reconciliation request to the same peer.
+ * This value allows to reconcile ~100 transactions (7 tx/s * 16s) during normal system operation at capacity.
+ * More frequent reconciliations would cause significant constant bandwidth overhead due to
+ * reconciliation metadata (sketch sizes etc.), which would nullify the efficiency.
+ * Less frequent reconciliations would introduce high transaction relay latency.
+ */
+static constexpr std::chrono::microseconds RECON_REQUEST_INTERVAL{std::chrono::seconds{16}};
+/**
+ * Interval between responding to peers' reconciliation requests.
+ * We don't respond to reconciliation requests right away because that would enable monitoring
+ * when we receive transactions (privacy leak).
+ */
+static constexpr std::chrono::microseconds RECON_RESPONSE_INTERVAL{std::chrono::seconds{2}};
+/** The size of the field, used to compute sketches to reconcile transactions (see BIP-330). */
+static constexpr unsigned int RECON_FIELD_SIZE = 32;
+static_assert(RECON_FIELD_SIZE % 8 == 0, "Field size should be divisible by 8");
```

Comment:
Why is this strictly needed? (apart from the BIP specifying that it is 32, so any other value is protocol breaking).